src/app_service/service/activity_center/dto/notification.nim

```diff
@@ -125,6 +126,11 @@ proc toActivityCenterNotificationDto*(jsonObj: JsonNode): ActivityCenterNotifica
   if jsonObj.contains("replyMessage") and jsonObj{"replyMessage"}.kind != JNull:
     result.replyMessage = jsonObj{"replyMessage"}.toMessageDto()
 
+  if jsonObj.contains("albumMessages") and jsonObj{"albumMessages"}.kind != JNull:
+    let jsonAlbum = jsonObj{"albumMessages"}
```

Comment:
```suggestion
    let jsonAlbum = jsonObj["albumMessages"]
```

Reply:
All calls to `jsonObj` here with `{}`. What's difference?

Reply:
It's not much. `{}` means "this might be not there` and `[]` this property is for sure there (it will throw if it's not). Since we already confirmed above that the property is there, we can use `[]`. I don't think it changes much, it's just a little detail that can make dev easier, because if you see the `[]`, you can assume it's there.